cpuid.c

```diff
@@ -11,65 +11,59 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 */
 
 /* Written by Martin Dimitrov, Carl Strickland */
+
 #include <stdio.h>
 
 #include "cpuid.h"
 
-
-void
-cpuid(uint32_t eax_in, uint32_t ecx_in,
-      cpuid_info_t *ci)
+void cpuid(uint32_t eax_in, uint32_t ecx_in, cpuid_info_t *ci)
 {
-    asm (
-#if defined(__LP64__)           /* 64-bit architecture */
-        "cpuid;"                /* execute the cpuid instruction */
-        "movl %%ebx, %[ebx];"   /* save ebx output */
-#else                           /* 32-bit architecture */
-        "pushl %%ebx;"          /* save ebx */
-        "cpuid;"                /* execute the cpuid instruction */
-        "movl %%ebx, %[ebx];"   /* save ebx output */
-        "popl %%ebx;"           /* restore ebx */
+    asm(
+#if defined(__LP64__)	 /* 64-bit architecture */
+	"cpuid;"	      /* execute the cpuid instruction */
+	"movl %%ebx, %[ebx];" /* save ebx output */
+#else			      /* 32-bit architecture */
+	"pushl %%ebx;"	/* save ebx */
+	"cpuid;"	      /* execute the cpuid instruction */
+	"movl %%ebx, %[ebx];" /* save ebx output */
+	"popl %%ebx;"	 /* restore ebx */
 #endif
-        : "=a"(ci->eax), [ebx] "=r"(ci->ebx), "=c"(ci->ecx), "=d"(ci->edx)
-        : "a"(eax_in), "c"(ecx_in)
-    );
+	: "=a"(ci->eax), [ebx] "=r"(ci->ebx), "=c"(ci->ecx), "=d"(ci->edx)
+	: "a"(eax_in), "c"(ecx_in));
 }
 
-cpuid_info_t
-get_vendor_signature()
+cpuid_info_t get_vendor_signature()
 {
     cpuid_info_t info;
     cpuid(0, 0, &info);
     return info;
 }
 
-uint32_t
-get_processor_signature()
+uint32_t get_processor_signature()
 {
     cpuid_info_t info;
     cpuid(0x1, 0, &info);
     return info.eax;
 }
 
-cpuid_info_t
-get_processor_topology(uint32_t level)
+cpuid_info_t get_processor_topology(uint32_t level)
 {
     cpuid_info_t info;
     cpuid(0xb, level, &info);
     return info;
 }
 
-void cast_uint_to_str(char* out, uint32_t in)
+void cast_uint_to_str(char *out, uint32_t in)
 {
     int i;
     uint32_t mask = 0x000000ff;
     for (i = 0; i < 4; i++) {
-        out[i] = (char)((in & (mask<<(i*8))) >> (i*8));
+	out[i] = (char)((in & (mask << (i * 8))) >> (i * 8));
```

Comment:
It seems this formatter uses tabs? This would be quite bad, please change it to use only spaces.